tests/integration/pkg/hooks/api-gateway.go

```diff
@@ -69,7 +70,7 @@ var ApiGatewayCrTearDownScenarioHook = func(ctx context.Context, sc *godog.Scena
 	return ctx, nil
 }
 
-var ApplyAndVerifyApiGatewayCrSuiteHook = func() error {
+func ApplyAndVerifyApiGateway(scaleDownOathkeeper bool) error {
```

Comment:
I think for a an easier to understand API it might be better to not export this function.